indico/modules/events/registration/__init__.py

```diff
@@ -77,9 +78,22 @@ def _inject_event_header(event, **kwargs):
                 .order_by(db.func.lower(RegistrationForm.title))
                 .all())
 
-    participants = get_unique_published_registrations(event)
+    # A participant could appear more than once in the list in case he register to multiple registration form.
```

Comment:
Also, try to keep <=71 chars for lines containing just text (i.e. docstrings and comments) - it's usually easier to read than very long lines of text.